### ISSUE TITLE
What do I prepare for re-train ?

### ISSUE THREAD

**Issue by Sero8139** [MichalBusta/DeepTextSpotter#72]
@MichalBusta
Hi,
I have been reading your PhD thesis , and I want to re-train the model using my own data .
but I don't know the data format for the input.
Can you a simple example for data format , and the process which you have trained ?
thanks.

**Comment by MichalBusta** [comment]
#53

I'd recommend to use our later work: https://github.com/MichalBusta/E2E-MLT
 - better framework (pytorch)
 - full python (easy to read)
 - better loss functions (=faster/stable training)
